mreg/signals.py

```diff
@@ -51,9 +51,11 @@ def _signal_history(resource, name, action, model, model_id, data):
                       model=model,
                       action=action,
                       data=data)
+
+    # We should never fail at performing a clean on the testdata itself.
```

Comment:
Why refering to testdata in code?

Reply:
Thanks! Comment gone, I was in the mindset of being in the test-suite itself...